doc/test_messages_documentation.py

```diff
@@ -121,6 +121,8 @@ def get_expected_messages(stream: TextIO) -> MessageCounter:
             line = match.group("line")
             if line is None:
                 lineno = i + 1
+            elif line.startswith("+") or line.startswith("-"):
+                lineno = i + 1 + int(line)
```

Comment:
Thank you we needed that 👌 Good job on copying how it's done in functional tests too !

Reply:
I think the tailing newline example was broken, do you mind fixing it in this pull request directly ?

Reply:
Fixed!

Reply:
Thanks 👍